storagenode/gracefulexit/worker.go

```diff
@@ -151,7 +156,18 @@ func (worker *Worker) transferPiece(ctx context.Context, transferPiece *pb.Trans
 		return err
 	}
 
-	putCtx, cancel := context.WithCancel(ctx)
+	putCtx := ctx
+
+	if worker.minBytesPerSecond == 0 {
+		// set minBytesPerSecond to default 128B if set to 0
+		worker.minBytesPerSecond = 128 * memory.B
+	}
+	maxTransferTime := time.Duration(int64(time.Second) * originalHash.PieceSize / worker.minBytesPerSecond.Int64())
+	if maxTransferTime < worker.minDownloadTimeout {
+		maxTransferTime = worker.minDownloadTimeout
+	}
+	var cancel func()
+	putCtx, cancel = context.WithTimeout(ctx, maxTransferTime)
```

Comment:
Is there a reason why can't we just do ` putCtx, cancel := context.WithTimeout(ctx, maxTransferTime) ` here?

Reply:
oh I didn't see the colon lol, yeah should've just done that